free-programming-interactive-tutorials-en.md

```diff
@@ -279,6 +279,7 @@
 
 ### SQL
 
+* [Khan Academy](https://www.khanacademy.org/computing/computer-programming/sql)
```

Comment:
Could we please use the name of the tutorial as given by the website?
You can mention that it's Khan Academy after the link instead.

This will help avoid having lots of resources just named Khan Academy, or Khan Academy being named twice in the same section if two interactive tutorials are added.

```suggestion
* [Intro to SQL: Querying and managing data](https://www.khanacademy.org/computing/computer-programming/sql) - Khan Academy
```